specifyweb/specify/scoping.py

```diff
@@ -30,26 +29,12 @@ def accession(self):
 
     def agent(self): return self._simple_division_scope()
 
-    def borrow(self): return self._simple_collection_scope()
-
-    def collectingevent(self): return self._simple_discipline_scope()
+    def borrowagent(self): return Scoping(self.obj.agent)()
 
-    def collectionobject(self): return self._simple_collection_scope()
-
-    def conservdescription(self): return self._simple_division_scope()
+    def collectingeventattribute(self): return Scoping(self.obj.collectingevent)
```

Comment:
Why was this line added?
While it is valid to assume that collectingeventattribute is scoped to collectingevent, there are many other tables that should also be scoped in a similar way but aren't.

I.e, here is a full list of such tables and how they should be scoped - https://github.com/specify/specify7/blob/586c48d10f16509d8315cc0401cfafcf65b51e8a/specifyweb/frontend/js_src/lib/components/DataModel/__tests__/__snapshots__/specifyModel.test.ts.snap#L1029-L1217 (the list was autogenereated by the front-end  - https://github.com/specify/specify7/blob/586c48d10f16509d8315cc0401cfafcf65b51e8a/specifyweb/frontend/js_src/lib/components/DataModel/specifyModel.ts#L442-L523)

Reply:
Yes, I can remove that! Oops. 
The line was added because I was looking at the Specify 6 `Collectingeventattribute` table (CollectingEventAttrs) rather than the Specify 5 `Collectingeventattribute` table.  (As far as I know, the Specify 6 `...Attribute` (`...Attrs`) tables are not entirely functional, correct?

However, if we assume that the Attrs tables are correct: 
then when `Collectingeventattribute` table is passed as 'obj' during the creation of a Scoping object, it will return a Collection scoping because  `Collectingeventattribute` only has the `collectionmemberid` 'scoping' field/relationship when it should get the relationship to Collecting Event and get its scoping.

Reply:
ah, you are right

Reply:
still, feel free to look at the above two files I linked if you are looking for more unscoped tables that should be scoped or the opposite